### PR TITLE
grammar in comments

### PR DESCRIPTION
**Pull request by alenros**

### CODE REVIEW

File: src/Components/blazor/src/Microsoft.AspNetCore.Blazor/Hosting/BlazorWebAssemblyHost.cs [hunk 1]
```diff
@@ -4,12 +4,12 @@
 namespace Microsoft.AspNetCore.Blazor.Hosting
 {
     /// <summary>
-    /// Used to to create instances a Blazor host builder for a Browser application.
+    /// Used to create an instances of Blazor host builder for a Browser application.
```
Comment (natemcmaster) [inline src/Components/blazor/src/Microsoft.AspNetCore.Blazor/Hosting/BlazorWebAssemblyHost.cs]:
```suggestion
    /// Used to create an instance of Blazor host builder for a Browser application.
```

As long as we're fixing grammar, instances => instance :)

**Comment by mkArtakMSFT** [comment]
Closing as there has been no activity on this PR for quite a while.

**Comment by alenros** [comment]
Any reason why this isn't accepted?

**Comment by SteveSandersonMS** [comment]
The file in question was already updated: https://github.com/aspnet/AspNetCore/blob/master/src/Components/Blazor/Blazor/src/Hosting/BlazorWebAssemblyHost.cs

Thanks for submitting the PR though! Sorry we were slow applying it.

**Pull request closed without merging**